appium/webdriver/webdriver.py

```diff
@@ -400,6 +402,23 @@ def current_activity(self):
         """
         return self.execute(Command.GET_CURRENT_ACTIVITY)['value']
 
+    def wait_activity(self, ac, retry=10, interval=1):
```

Comment:
how about shorter `activity` instead of `app_activity`? And yes, `retries` is better than `retry`. Will `timeout` make it seems like to be the total time out setting?